class ValidationError(Exception):
    pass

class RepoError(Exception):
    pass